docs/UserGuide.md

```diff
@@ -71,20 +95,33 @@ Shows a message explaning how to access the help page.
 
 Format: `help`
 
+### Managing Patient
 
-### Adding a person: `add`
+#### Adding a patient: `add`
 
-Adds a person to the address book.
+Adds a patient and their relevant details to MediBase3.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
+
+<div markdown="block" class="alert alert-info">:information_source: **Note:** <br>
+
+* All fields are compulsory.
+* The NRIC serves as a unique identifier for each patient.
+* A patient will not be added if the NRIC given is already associated with another patient in MediBase3. An error message will be displayed in this case. 
+* Refer to the [Parameter Details](#Parameter-Details) section for more information on the constraints of each parameter.
 
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
 </div>
```

Comment:
(1) disable italics in `docs/_sass/minima/_base.scss`
![image](https://github.com/user-attachments/assets/d678fbc0-fa88-47e4-b9c0-8dc4447f447c)

(2)

convert 
```html
<div markdown="span" class="XYZ">
```
to 
```
{: .XYZ}
> block quote comments
```

Like this:
```
{: .p .alert .alert-info}
> **:information_source: Notes about the command format:**
>
> * Words in `UPPER_CASE` are the parameters to be supplied by the user.    
>   e.g. in `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`, `NAME` is a parameter which can be used as `n/John Doe`.
> * Items in square brackets are optional.    
>
>   e.g `edit NRIC [n/NAME] [i/NRIC] [g/GENDER] [d/DOB] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS]` can be used as `edit S1234567A n/John Lim g/M` or as `edit S1234567A g/M`.
>
> * Items with `…`​ after them can be used multiple times.    
>   e.g. `[c/CONDITION]…​` can be used as, `c/Knee Pain`, `c/Flu c/Fever` etc.
>
> * Parameters can be in any order.    
>   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
>
> * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
>   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
>
> * The command name and prefixes are case-sensitive.    
>   e.g. `add` is not the same as `Add`, `c/` is not the same as `C/`.
>
> * If you are using a PDF version of this document, be careful when copying and pasting commands that span multiple lines as space characters surrounding line-breaks may be omitted when copied over to the application.
```

![image](https://github.com/user-attachments/assets/8e971fd5-0b03-4e18-b580-19b12b85e8e6)